test/client/config/karma.conf.js

```diff
@@ -37,6 +40,15 @@ module.exports = config => {
             define: {
                 global: 'window',
             },
+            plugins: [
+                alias(
+                    Object.fromEntries(
+                        Object.entries( pkg.browser ).map( ( [ key, value ] ) => (
+                            [ key, path.resolve( cwd, `${value}.js` ) ]
+                        ) )
+                    )
+                ),
+            ],
```

Comment:
This change, as well as the one in `build.js`, addresses another media regression (and potentially several other regressions 😬), this time caused by #289:

1. [`enketo-core`](https://github.com/enketo/enketo-core/blob/53eaeb5e6533e302087e700b203725efbfeb1aad/package.json#L90-L97) defines and implements a set of aliased modules whose functionality it depends on
2. [`enketo-express`](https://github.com/enketo/enketo-express/blob/ca75e8eeb9481eddb695538949beaf419843a083/package.json#L123-L129) also defines those aliases, but overrides those implementations.
3. When we switched to esbuild, it failed to account for those overrides, causing the `enketo-express` build to use `enketo-core`'s behavior for those modules.

Reply:
We should consider consolidating relevant portions of these configs, so the two builds don't diverge in the future.